commcare_connect/opportunity/views.py

```diff
@@ -172,7 +172,7 @@ def get_queryset(self):
         opportunity = get_object_or_404(Opportunity, organization=self.request.org, id=opportunity_id)
         access_id = self.kwargs["pk"]
         access = get_object_or_404(OpportunityAccess, opportunity=opportunity, pk=access_id)
-        return Payment.objects.filter(opportunity_access=access)
+        return Payment.objects.filter(opportunity_access=access).order_by("date_paid")
```

Comment:
I think you want to do this in the opposite direction so the most recent payment is first

Reply:
Fixed. https://github.com/dimagi/commcare-connect/pull/233/commits/1080718825bd42758b3ffc0f05519e74cd96716c